user_guide_src/source/changelogs/v4.6.0.rst

```diff
@@ -108,6 +108,8 @@ Model
 Libraries
 =========
 
+- Added `retainMultiplePatterns()` to `FileCollection` class.
```

Comment:
Can you add a link to the section `retainMultiplePatterns` in the page `file_collections.rst`?
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/documentation.rst#to-a-section